book/src/05_unified_platform/01_catalyst-cardano-bridge/05_database/03_transaction_state.md

```diff
@@ -0,0 +1,26 @@
+# Transaction State Database
+
+This is NOT a time-series database, it tracks:
+
+* the current state of the sync from the block chain
+* all current UTXO's
+* any other data it needs to calculate staked-ADA changes as blocks arrive.
+
+It is updated to track current state and is not historical.
+
+This state is updated atomically, along with:
+
+* The staked ADA database
+* The registration database
+
+This ensures that the DB is always in-sync with discreet minted blocks on the block-chain.
```

Comment:
typo: discrete (discreet is the "private" one)

Reply:
We can't have `discreet` minted blocks until Cardano brings in zksnarks on the mainnet.  That's true.

Reply:
fixed